src/fheroes2/heroes/heroes_dialog.cpp

```diff
@@ -317,7 +317,7 @@ int Heroes::OpenDialog( const bool readonly, const bool fade, const bool disable
 
     StatusBar statusBar;
     // Status bar must be smaller due to extra art on both sides.
-    statusBar.setRoi( { dst_pt.x + 16, dst_pt.y + 3, bar.width() - 16 * 2, 0 } );
+    statusBar.setRoi( { dst_pt.x + 16, dst_pt.y, bar.width() - 16 * 2, 0 } );
```

Comment:
Comparing the status bar in hero dialog to the others we may notice that the text is one pixel higher.
To make the rendering better we can move it one pixel down:
```suggestion
    statusBar.setRoi( { dst_pt.x + 16, dst_pt.y + 1, bar.width() - 16 * 2, 0 } );
```

Reply:
Great eye!!!

<img width="550" alt="Screenshot 2025-04-09 at 2 26 44 PM" src="https://github.com/user-attachments/assets/c77d2a09-a087-4ed2-a160-7fc272d35a1e" />